schemas/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mozilla-nimbus-schemas"
-version = "2024.11.5"
+version = "2024.12.1"
 description = "Schemas used by Mozilla Nimbus and related projects."
 authors = ["mikewilli"]
```

Comment:
Unrelated to PR, but probably worth changing this... #11898

Reply:
in another patch, on a crunch